### ISSUE TITLE
Warnings in swagger.json

### ISSUE THREAD

**Issue by davidsneighbour** [Luracast/Restler#635]
My restler explorer is only working, if I set error_reporting to 0 before loading one specific class (amongst many). If not, it throws two warnings, that do not say much in sense of what is going wrong. The code that is "creating" the warnings is the following method:

```
    /**
     * @url POST /create/
     *
     * @param array<array-key, mixed> $request_data
     *
     * @return string|false
     */
    public static function postCreate(array $request_data = [])
    {
        try {
            $tour = Tours2::post(
                array_merge($request_data, ['is_charter' => '1'])
            );

            return '/bookings/add/?tripID=' . $tour['id'];
        } catch (GetParameterMissing $eException) {
            Tracking::trackException($eException);
        } catch (Exception $eException) {
            Tracking::trackException($eException);
        }

        return false;
    }
```

The API method itself works as expected without thrown exceptions or warnings, only the Explorer refuses to load the methods. In swagger.json the following warnings come up (sorry image, i can't copy the warnings properly):

![Selection_015](https://user-images.githubusercontent.com/83281/87678255-eba14480-c7a4-11ea-9b21-b96fdab669aa.png)

The resulting json after the warnings is properly formatted and has no errors. 

index:146 is `$oRestler->handle();`

Where should I look for the error?

**Comment by davidsneighbour** [comment]
It get's weirder. If I suppress the warnings then the explorer expects string as parameter for POST:

![Selection_016](https://user-images.githubusercontent.com/83281/87680145-4176ec00-c7a7-11ea-9b6d-a6432dd2831d.png)

**Comment by Arul-** [comment]
Can you try changing the comment as follows

    @param array $request_data {@type associative}

Instead and try!

**Comment by davidsneighbour** [comment]
The error stays. (I deleted the cache directory too.) I wonder if there is some deeper issue that results in it trying to be a string, but receiving array... chaos :)

Using the 3.1.0 tag by the way.

**Comment by davidsneighbour** [comment]
the json created for this function is 

```
        "/tours/charter/create": {
            "post": {
                "operationId": "charterCreateCreate",
                "tags": [
                    "tours"
                ],
                "parameters": [
                    {
                        "name": "charterCreateCreateModel",
                        "description": "request_data  \n",
                        "in": "body",
                        "required": false,
                        "schema": {
                            "$ref": "#/definitions/charterCreateCreateModel"
                        }
                    }
                ],
                "summary": " 🔓",
                "description": "",
                "responses": {
                    "200": {
                        "description": "Success",
                        "schema": {
                            "type": "string"
                        }
                    }
                }
            }
        }
```

**Comment by Arul-** [comment]
Noted

**Comment by Arul-** [comment]
Fixed in 8f038ef6bd75a7ea1fc2b5abe186bfb2cc1887b2 in V5 branch. The issue is because multiple return types are not supported in Explorer.  Fix assumes the last one in the list as the type for Explorer's consideration

In your above example, it is `boolean`

Please test and let me know!

**Comment by Arul-** [comment]
Now available in 5.0.7
